plugins/in_elasticsearch/in_elasticsearch_bulk_prot.h

```diff
@@ -0,0 +1,39 @@
+/* -*- Mode: C; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- */
+
+/*  Fluent Bit
+ *  ==========
+ *  Copyright (C) 2015-2023 The Fluent Bit Authors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+#ifndef FLB_IN_ELASTICSEARCH_BULK_PROT
+#define FLB_IN_ELASTICSEARCH_BULK_PROT
+
+#define ES_VERSION_RESPONSE "{\"version\":{\"number\":\"8.0.0\",\"build_flavor\":\"fluent-bit OSS\"},\"tagline\":\"fluent-bit's Bulk API compatible endpoint\"}"
```

Comment:
when naming the agent pls use `Fluent Bit` instead of `fluent-bit` (binary name)

Reply:
Fixed in https://github.com/fluent/fluent-bit/pull/6685/commits/0fa14eb9e3ceedd0cc962c1200cea2c03d26968e